packages/payment-integration-api/src/address/index.ts

```diff
@@ -1 +1,2 @@
 export { default as Address, AddressKey, AddressRequestBody } from "./address";
+export { default as InternalAddress } from "./internal-address";
```

Comment:
Do we still need this? In theory we should be deleting this folder completely?

Reply:
It uses in chain by SubmitPaymentAction